### PR TITLE
Model updates; attributes removal; features removed

### PR DESCRIPTION
**Pull request by cecemel**
some updates on the model lead to removal of feaures

Do we get rid of https://github.com/lblod/frontend-loket/blob/development/app/components/public-services/sector-select.hbs or do we keep it as an example, because probably we will need similar things on other selectors

### CODE REVIEW

File: app/routes/public-services/index.js [hunk 1]
```diff
@@ -23,23 +19,16 @@ export default class PublicServicesIndexRoute extends Route {
...
-      sector,
     };
   }
 
   @restartableTask
-  *loadPublicServicesTask({ search, sector, page, sort }) {
+  *loadPublicServicesTask({ search, page, sort }) {
     let query = {
       'page[number]': page,
-      include: 'sectors,status,lifecycle-status',
```
Comment (Windvis) [inline app/routes/public-services/index.js:42]: I think we still want to include the `status` relationship since it's shown in the table?

Comment (cecemel) [reply app/routes/public-services/index.js:42]: He sees it all :-) fixing it

File: app/components/public-services/status.js [hunk 2]
```diff
@@ -1,34 +1,22 @@
...
-    'Ontwerp',
-  'http://data.lblod.info/id/concept/ipdc-thema/27d784f8-e7ea-4f18-88f8-02d427033009':
-    'Actief',
-  'http://data.lblod.info/id/concept/ipdc-thema/12873bfb-d2cc-4cc5-9255-1bf0ca2747f9':
-    'Stopgezet',
-};
+const DOCUMENT_STATUS = {
+  'http://lblod.data.gift/concepts/79a52da4-f491-4e2f-9374-89a13cde8ecd': {
+    skin: 'warning',
+    label: 'ontwerp',
```
Comment (Windvis) [inline app/components/public-services/status.js:6]: It's intended that these labels no longer start with an uppercase letter?

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
There is still a linting error but I think it's resolved by the other PR so might not be worth fixing for this one?

File: app/templates/public-services/add.hbs [hunk 3]
```diff
@@ -90,8 +75,6 @@
       {{else if this.hasResults}}
         <Content.body as |publicService|>
           <td>{{! template-lint-disable no-triple-curlies~}} {{{publicService.nameNl}}}</td>
-          <td>{{publicService.identifier}}</td>
```
Comment (Windvis) [inline app/templates/public-services/add.hbs:93]: oh, I think we can also remove the attr from the model?

**Comment by cecemel** [comment]
> There is still a linting error but I think it's resolved by the other PR so might not be worth fixing for this one?

yeah about a hasMany import? Should be fixed in the following pr

**Pull request merged**